### PR TITLE
meta: fix case of extensions on internal docs and move CoC to admin repo

### PR DESCRIPTION
**Pull request by MylesBorins**
MD -> md

### CODE REVIEW

**Comment by MylesBorins** [comment]
While doing this I have confirmed that both the CoC + moderation policy currently have no delta from the current files in the TSC repo

**Comment by Trott** [comment]
There are probably a lot of places that link directly to these documents in their existing locations.  I guess they won't *need* updating if the info exists in two places or if the current place has a pointer to the new place. Still, definitely worth updating what you can find easily. Some of this stuff is in non-obvious places. For example, there's a link to the Moderation policy in the description of the @-nodejs/moderation team.

**Comment by MylesBorins** [comment]
@Trott I started reviewing what needed to be updated and am willing to go through the entire org to update all projects, just don't want to do that work until I know we have sign off on this

**Comment by MylesBorins** [comment]
For additional context @jasnell was -1 on moving the Code of Conduct out of the TSC repo. While I don't agree he does raise some valid points. This seems like something that both the TSC + CommComm should discuss

https://github.com/nodejs/TSC/pull/395#issuecomment-339017773

**Comment by bnb** [comment]
@MylesBorins added cc-agenda label for review in the meeting in < 12 hours. 👍

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+To learn more about the process of how moderators enforce the Code of Conduct, 
+review the [Moderation Policy] and [Requesting Moderation].
+
+[Moderation Policy]: https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md#moderation-policy
```
Comment (thefourtheye) [inline CODE_OF_CONDUCT.md]: These should go to the bottom of the page?

**Comment by bnb** [comment]
Going to remove the cc-agenda label since we talked about this in last week's meeting. @nodejs/community-committee please review if you've not already ❤️

**Pull request merged**